templates/.circleci/config.yml

```diff
@@ -166,7 +166,7 @@ jobs:
             echo 'export AWS_ECR_ACCOUNT_URL=<< parameters.account-id >>.dkr.ecr.<< parameters.region >>.amazonaws.com' >> $BASH_ENV
       - aws-ecr/build-and-push-image:
           repo: << parameters.repo >>
-          tag: $VERSION_TAG,latest
+          tag: $VERSION_TAG,latest,last-deployed
```

Comment:
We shouldn't be tagging this here. It should only happen after the deploy to staging. Otherwise something may have failed in a test or deploy step meaning the image wasn't deployed, but the tag would still be updated.

Reply:
done at Deploy process when successful